redwood-treehouse-host/src/androidMain/kotlin/app/cash/redwood/treehouse/TreehouseLayout.kt

```diff
@@ -57,6 +57,12 @@ public class TreehouseLayout(
 
   override var saveCallback: TreehouseView.SaveCallback? = null
 
+  init {
+    // The view needs to have an id for Android to populate saved data back
```

Comment:
nit:
```suggestion
    // The view needs to have an id for Android to populate saved data back.
```
[Comments should be sentences](https://nedbatchelder.com/blog/201401/comments_should_be_sentences.html).